<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractionPanel</class>
 <widget class="QWidget" name="InteractionPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="OpenGraph">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Open graph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Redraw">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>840</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Redraw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DFS">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>DFS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BFS">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>BFS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DFS_Input">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>83</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="BFS_Input">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>154</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="GetShortestPathBetweenVertices">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>270</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Shortest Path Between Vertices</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetShortestPathsBetweenAllVertices">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>400</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Shortest Paths Between All Vertices</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TSM">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>780</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Traveling</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetShortestPathsBetweenAllVertices_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>210</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Least Spanning Tree</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>470</y>
     <width>470</width>
     <height>291</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Vertex</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Path</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="BFS_Input_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>340</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="BFS_Input_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
